ui/drivers/ui_cocoa.m

```diff
@@ -921,7 +927,12 @@ static void ui_companion_cocoa_deinit(void *data)
 static void *ui_companion_cocoa_init(void) { return (void*)-1; }
 static void ui_companion_cocoa_notify_content_loaded(void *data) { }
 static void ui_companion_cocoa_toggle(void *data, bool force) { }
-static void ui_companion_cocoa_event_command(void *data, enum event_command cmd) { }
+static void ui_companion_cocoa_event_command(void *data, enum event_command cmd)
+{
+   id block = [^{ command_event(cmd, data); } copy];
```

Comment:
Hi, this PR is awesome on the surface, just one question though, can we do without the lambda here? This would kill off backwards compatibility with older Xcode versions that would not yet support lambdas (think PowerPC compilers circa 10.5, etc).

Reply:
Hi.
Well, we can replace the lambda with an object of a custom class. No so succinctly but it works. What do you think?